src/components/Modal/Modal.js

```diff
@@ -18,9 +19,9 @@ class Modal extends React.Component {
      */
     title: PropTypes.string,
     /**
-     * Handles click events on the backdrop behind the modal. Use this to close it if desired.
+     * Handles click events on the backdrop behind the modal or on the X in the title bar.
      */
-    onBlockerClicked: PropTypes.func,
+    onClose: PropTypes.func,
```

Comment:
We should mark `onClose` as required.

Reply:
Some modals are closed by internal button controls, which would force the user to pass a no-op to the modal component itself.